### PR TITLE
fix: #813

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes/no
| New feature?     | yes/no
| BC breaks?       | yes/no
| Deprecations?    | yes/no
| New tests added? | yes/not needed
| Fixed tickets    | #813
| License          | MIT

### Description

[Description of the bug or feature]

--

### CODE REVIEW

File: src/muya/lib/utils/exportHtml.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ class ExportHtml {
    * @param {*} title Page title
    * @param {*} printOptimization Optimize HTML and CSS for printing
    */
-  async generate (title = '', printOptimization = false) {
+  async generate (title = '', printOptimization = false, extraCss = '') {
```
Comment (fxha) [inline src/muya/lib/utils/exportHtml.js]: For what is `extraCss` used for?

File: src/muya/lib/eventHandler/clickEvent.js [hunk 2]
```diff
@@ -25,11 +25,29 @@ class ClickEvent {
...
-      // Commit native cursor position because right-clicking doesn't update the cursor postion.
-      contentState.cursor = {
-        start,
-        end
+      if (
+        nextTextBlock && nextTextBlock.key === end.key &&
+        end.offset === 0 &&
+        start.offset === startBlock.text.length
+      ) {
+        // Set cursor at the end of start Block. and reset cursor
```
Comment (fxha) [inline src/muya/lib/eventHandler/clickEvent.js]: Typo `Block. and`

File: src/renderer/contextMenu/editor/index.js [hunk 3]
```diff
@@ -35,8 +35,8 @@ export const showContextMenu = (event, { start, end }) => {
       SEPARATOR
     )
   }
-
-  [CUT, COPY, COPY_AS_HTML, COPY_AS_MARKDOWN].forEach(item => {
+  console.log(disableCutAndCopy)
```
Comment (fxha) [inline src/renderer/contextMenu/editor/index.js]: Please remove debug code.

File: src/muya/lib/utils/exportHtml.js (hunk 1 above)
Comment (Jocs) [reply src/muya/lib/utils/exportHtml.js]: We can select theme or adjust the padding and page size when we export HTML, so I add this `extraCss`.

**Pull request merged**